front/src/common/BootstrapSNCF/NavBarSNCF.tsx

```diff
@@ -24,17 +25,25 @@ type Props = {
   logo?: string;
 };
 
-const LegacyNavBarSNCF = ({ appName, logo = getLogo() }: Props) => {
+const LegacyNavBarSNCF = ({ appName, logo }: Props) => {
   const { openModal } = useModal();
+  const { digitalTwin, customizedDeployment } = useLogo();
   const safeWord = useSelector(getUserSafeWord);
   const { t } = useTranslation('home/navbar');
   const { logout, username } = useAuth();
 
+  const headerLogo = logo ?? digitalTwin?.logo;
```

Comment:
`digitalTwin` is always defined since you initialized in `useLogo`. 
- you can remove the `?` 
- you can remove the test line 41 since logo will always be defined
- I think this const is not needed, you can directly use the image like this:
  `<img src={logo ?? digitalTwin.logo} />`
```suggestion
  const headerLogo = logo ?? digitalTwin.logo;
```

Reply:
Done